PolyDeploy/SqlDataProviders/00.09.00.SqlDataProvider

```diff
@@ -0,0 +1,245 @@
+﻿/************************************************************/
+/***** SqlDataProvider                                  *****/
+/*****                                                  *****/
+/*****                                                  *****/
+/***** Note: To manually execute this script you must   *****/
+/*****       perform a search and replace operation     *****/
+/*****       for {databaseOwner} and {objectQualifier}  *****/
+/*****                                                  *****/
+/************************************************************/
+/*** Cantarus_PolyDeploy v0.9.0                           ***/
+/************************************************************/
+
+/*********************************************************************************************************/
+/*** Cantarus_PolyDeploy_DropUQConstraints STORED PROCEDURE **********************************************/
+/***                                                                                                   ***/
+/*** Add stored procedure for dropping unnamed unique constraints.                                     ***/
+/*********************************************************************************************************/
+
+IF EXISTS (SELECT * FROM dbo.sysobjects WHERE id = object_id(N'{databaseOwner}[{objectQualifier}Cantarus_PolyDeploy_DropUQConstraints]'))
+    DROP PROCEDURE {databaseOwner}[{objectQualifier}Cantarus_PolyDeploy_DropUQConstraints]
+GO
+
+/*
+    This procedure allows the easy deletion of unnamed unique constraints.
+
+    When a constraint is created without a name being specified SQL server will
+    generate a random name for the constraint based on the constraint type, the
+    table name and a random number. It's difficult to drop these constraints
+    using statements as the constraint will have a different name on every
+    installation.
+
+    This procedure allows you to pass a table name (without the
+    'Cantarus_PolyDeploy_' prefix) and a column name within that table. It will
+    look up the name of the unique constraint (if there is one) and
+    automatically execute the statement required to drop the constraint.
+
+    Note: The second AND statment within the WHERE clause makes use of the LEFT
+    operator. This is because when the database owner token is replaced it
+    includes the trailing '.'. Using LEFT with CHARINDEX allows us to use the
+    database owner token to match the appropriate schema.
+*/
+CREATE PROCEDURE {databaseOwner}[{objectQualifier}Cantarus_PolyDeploy_DropUQConstraints]
+    @TableName [VARCHAR](64),
+    @ColumnName [VARCHAR](32)
+AS
+    DECLARE @ConstraintName [VARCHAR](128)
```

Comment:
Please add semicolons to the end of each SQL Statement, like so:
``` sql
DECLARE @ConstraintName [VARCHAR](128);

SET @ConstraintName = (...);

IF (@ConstraintName IS NOT NULL)
    BEGIN
        EXEC('...');
    END
```

It is particularly unclear to read later on in this file where you're doing several `ALTER TABLE` clauses within a conditional block.